pkg/consts/consts.go

```diff
@@ -0,0 +1,16 @@
+package consts
+
+const UserName = "username"
```

Comment:
There are a few types of consts in this file, let's separate them (there are consts here that or "config" so IMO they should be in a separate pkg